content/contributing/development/git_guidelines.rst

```diff
@@ -0,0 +1,142 @@
+=================
+Git guidelines
+=================
+
+Configure your git
+------------------
+
+Based on ancestral experience and oral tradition, the following things go a long
+way towards making your commits more helpful:
+
+- Be sure to define both the user.email and user.name in your local git config
+
+  .. code-block:: text
+
+     git config --global <var> <value>
+
+- Be sure to add your full name to your Github profile here. Please feel fancy
+  and add your team, avatar, your favorite quote, and whatnot ;-)
```

Comment:
Maybe a good opportunity to enforce a github tag pattern ? :eyes:

Reply:
I'm not updating the Git guidelines in this PR. I will only refresh the RST.

Reply:
ah okay